docs/development-environment.md

```diff
@@ -40,25 +40,25 @@ In order to test the provider, you can run `make test`.
 
 *Note:* Make sure no `AWS_ACCESS_KEY_ID` or `AWS_SECRET_ACCESS_KEY` variables are set, and there's no `[default]` section in the AWS credentials file `~/.aws/credentials`.
 
-```sh
+```console
 $ make test
```

Comment:
above, what do you think if we added `unit` to the line e.g. `In order to [unit] test the provider, you can run `make test`. for specificity?